docs/reference/query-dsl/script-query.asciidoc

```diff
@@ -25,18 +29,29 @@ GET /_search
         }
     }
 }
-----------------------------------------------
+----
 // CONSOLE
 
-[float]
-==== Custom Parameters
 
-Scripts are compiled and cached for faster execution. If the same script
-can be used, just with different parameters provider, it is preferable
-to use the ability to pass parameters to the script itself, for example:
+[[script-top-level-params]]
+==== Top-level parameters for `script`
+
+`script`::
+(Required, <<modules-scripting-using, script object>>) Contains a script to run
+as a query.
```

Comment:
May be worth to add that the script must return a boolean value, `true` or `false`.

Reply:
Added with 021a9cc.